File: src/Lession4/Ex1.java
package Lession4;

import java.util.Scanner;

public class Ex1 {
    static void main() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String str = sc.next();

        System.out.println(capitalize(str));
    }

    static String capitalize (String inputString ) {
        char first = inputString.charAt(0);

        if (first > 'Z') {
            first -= 'a' - 'A';
        }

        return first + inputString.substring(1);
    }
}

File: src/Lession4/Ex1.kt
package Lession4

fun main() {
    print("Enter a string: ");
    val str = readln();
    print(capitalize(str))
}

fun capitalize (inputString: String): String {
    var first = inputString[0];
    if (first > 'Z') {
        first -= 'a' - 'A'
    }
    return first + inputString.substring(1);
}